src/index.js

```diff
@@ -0,0 +1 @@
+require('../vendor/aframe.min.js');
```

Comment:
does npm keep up to date with the latest version? 

I've had trouble in the past where certain other modules are only compatible with a particular version, sometimes master, and so it was always easier to just download the latest and include it in case i couldn't get it from npm, which is why i tend to default to this.

what's your experience been like using npm for aframe, sufficient?